config/default.json

```diff
@@ -108,5 +108,13 @@
       "encryptionKey": "not-a-real-32-bit-encryption-key",
       "AppInsightsInstrumentationKey":"APPINSIGHTS_INSTRUMENTATIONKEY"
     }
+  },
+  "launchDarkly": {
+    "featureFlag-keys": {
+      "admissions": "cmc_admissions"
+    },
+    "sdk-keys": {
```

Comment:
you don't need different keys per env, just one and use an environment variable to bring it in, see 
https://github.com/hmcts/cmc-citizen-frontend/blob/master/config/custom-environment-variables.json

Reply:
Move config to ENV variable and ADD to terraform